### PR TITLE
feat: Besu configurations + max-peers flags adjustment

### PR DESCRIPTION
**Pull request by Wolmin**
**Changelog:**

1. This PR provides configuration files for Besu client - https://www.hyperledger.org/projects/besu
2. This PR also changes the max-peers settings to the client's defaults.

### CODE REVIEW

File: mainnet/besu/besu.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+'data-path' = './mainnet-data/execution'
+'genesis-file' = './configs/mainnet/shared/genesis.json'
+'network-id' = 42
+'engine-jwt-secret' = './configs/shared/secrets/jwt.hex'
+'rpc-http-enabled' = true
+'rpc-http-port' = 8545
```
Comment (mxmar) [inline mainnet/besu/besu.toml:6]: Is it `localhost` host by default?

File: mainnet/shared/genesis.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "config": {
     "chainId": 42,
+    "ethash": {},
```
Comment (mxmar) [inline mainnet/shared/genesis.json:4]: Do `Geth` and `Erigon` able to load this config with`"ethash": {},`?

File: mainnet/besu/besu.toml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+'data-path' = './mainnet-data/execution'
+'genesis-file' = './configs/mainnet/shared/genesis.json'
+'network-id' = 42
```
Comment (mxmar) [inline mainnet/besu/besu.toml:3]: How can we set our `miner.gasprice` according to Geth or Erigon for example?

File: mainnet/besu/besu.toml (hunk 1 above)
Comment (Wolmin) [reply mainnet/besu/besu.toml:7]: No, its null, Ive adjusted that

**Review by mxmar — APPROVED** [review]

**Pull request merged**